### DISCUSSION TITLE
colour : Blue/orange vs orange-red / blue-green

### DISCUSSION THREAD

**Discussion by JeffAbrahamson** [community/community#10676] [Codespaces]
The - and + are way more distinct now with the colour-deficient mode, but there’s no cultural association with blue/orange the way there is with red/green.

With traffic lights, the solution has been to push red lights towards orange and green lights towards blue. This separates them for colour deficient people but those without red/green deficiency rarely notice.

May I suggest you consider making the default red a bit more orange, the default green a bit more blue, and perhaps you’ll have something, like traffic lights, that doesn’t require choosing a special viewing option to work optimally.

---

This discussion duplicates [this one](https://github.community/t/blue-orange-vs-orange-red-blue-green/222917?u=jeffabrahamson) as requested by @liligalante (staff).

**Comment by asciimike** [comment]
@JeffAbrahamson thanks for the feedback. I'll agree that colors are generally problematic for a number of reasons and we'll never get one true color palette that meets everyone's needs. Since the diff colors are controlled by VSCode (not Codespaces), you can [change the theme](https://code.visualstudio.com/docs/getstarted/themes), install an extension, or call the APIs yourself (e.g. https://code.visualstudio.com/api/references/theme-color#diff-editor-colors).
